### DISCUSSION TITLE
How to change Middle Mouse Button to Left Mouse Button for pausing?

### DISCUSSION THREAD

**Discussion by Xiarno** [mpv-player/mpv#14266] [Q&A]
I am trying to switch the Middle Mouse Button to Left Mouse Button to pause the video player. However, it doesn't work at all for some reason.

I've used the input configurator to try to do that without success.
![15123_mpvnet_JsAzTEAO2y](https://github.com/mpv-player/mpv/assets/24660485/43ea5947-3014-486f-8cab-b83694291fa5)

How can I do it please? I wanna keep all the other option for pausing too (Like spacebar).
And no I do not care about the issue of dragging with left click anywhere on the screen (That's mostly the thing I've found when searching for a solution on this problem :/)
Thanks!

**Comment by Xiarno** [comment]
I just realized there's this thing that is there for some reason. Could it be the reason that it isn't working at all? I am unable to unassign this "ignore". Each time I try to remove it, it comes back and I don't understand why.
![15124_mpvnet_Zpb38x9Ra6](https://github.com/mpv-player/mpv/assets/24660485/bb234713-84f8-4d9d-b22c-be44c4cf4b38)

**Comment by llyyr** [comment]
Ask for mpv.net support on https://github.com/mpvnet-player/mpv.net/

**Reply by Xiarno** [reply]
Okay.

**Comment by orhantopac** [comment]
I have an easier work around copy and paste this to Command of ignore without quotes
"script-message-to mpvnet play-pause"
Close and re open mpv.net  and there is the ignored functionality.

If you delete "ignore" command, it reverts vack because it is hardcoded....
If you (let's say) change space or MBTN_MDL to MBRTN_LEFT it will still ignore.
But it may count as a bug but, if you do my solution, it will add this line to the input.conf
`MBTN_Left   script-message-to mpvnet play-pause  # Ignore left mouse butten`

When you start program, it will firstly create hard-coded "ignore" object
but secondly it will create a second "Ignore the left mouse butten" so this do and will override the first ignore.
Strangely enough this duplication as my 5 minute try, only happens when you assign it with play/pause, not only it won't work it will duplicate 3 or 4 input rules with MBTN_LEFT that won't work, since "ignore" is on the work, since my workaround uses "ignore" itself, it ignores "Ignore left mouse butten" rule/object what ever you call.

![image](https://github.com/user-attachments/assets/a4d035d6-3ec7-40e8-b59b-5e6bb4d9f398)
![image](https://github.com/user-attachments/assets/8bc78402-c152-4043-9dfd-3898bb426686)
